docs/sources/k6/next/extensions/run-extensions-using-binary-provisioning.md

```diff
@@ -0,0 +1,145 @@
+---
+title: 'Run extensions using Binary Provisioning'
+description: 'Guide to run scripts that require extensions using Binary provisioning.'
+weight: 04
+---
+
+# Run extensions using Binary Provisioning
+
+{{< admonition type="caution" >}}
+
+This is an experimental feature. Breaking changes might occur prior to the feature being made generally available.
+
+{{< /admonition >}}
+
+k6 now supports natively running scripts that use extensions using the Binary Provisioning feature.
+
+When enabled, this feature will automatically provide a custom binary including all the required extensions and will execute the script using this binary.
+
+Currently, it only supports extensions [supported in Grafana Cloud](https://grafana.com/docs/grafana-cloud/testing/k6/author-run/use-k6-extensions/#supported-extensions-in-grafana-cloud).
+
+If your test requires other extensions, you can create a [custom k6 binary](https://grafana.com/docs/k6/<K6_VERSION>/extensions/#xk6-makes-custom-binaries).
+
+The feature requires authentication to Grafana Cloud and therefore is supported only in the `cloud run` command.
+
+Running scripts locally with the `run` command will not use Binary Provisioning, but you can use `cloud run --local-execution` instead.
+
+The `archive` and `inspect` commands also support Binary provisioning to allow creating archives to the cloud.
+
+To enable the Binary Provisioning, you must set the `K6_BINARY_PROVISIONING` environment variable to `true`:
+
+{{< code >}}
+
+```linux
+export K6_BINARY_PROVISIONING=true
+```
+
+```mac
+export K6_BINARY_PROVISIONING=true
+```
+
+```windows-powershell
+$Env:K6_BINARY_PROVISIONING = "true"
+
+```
+
+```windows
+```
+
+{{< /code >}}
+
+
+In order to use the Binary Provisioning, the user must be authenticated to Grafana Cloud. You can [authenticate](https://grafana.com/docs/grafana-cloud/testing/k6/author-run/tokens-and-cli-authentication/#authenticate-with-the-login-command) using the `k6 cloud login` command:
+
+{{< code >}}
+
+```linux
+k6 cloud login --token <API_TOKEN>
+```
+
+```mac
+k6 cloud login --token <API_TOKEN>
+```
+
+```windows-powershell
+
+```
+
+```windows
+```
```

Comment:
```suggestion
```

Can we remove these since there are no specific commands for it?

Reply:
@heitortsergent I think there should be, because this is available for Windows users, only that I don't have a windows machine to test them